packages/web-components/src/global/_variables.scss

```diff
@@ -28,6 +28,14 @@
     --normal-border: #56f000;
     --off-fill: #a4abb6;
     --off-border: #a4abb6;
+
+    //focus variables
+    --focus-ring-color: var(--color-palette-pink-200);
```

Comment:
just need to update to use the latest tokens. may need to rebase main to pull the latest if they arent in here 

--color-border-focus-default - The color of a focused element’s outline
--border-width-focus-default The thickness of a focused element’s outline
--spacing-focus-default  - The amount of space between a focused element’s outline and edge

Reply:
All set!